pkg/registry/core/pod/storage/eviction.go

```diff
@@ -128,9 +128,17 @@ func (r *EvictionREST) Create(ctx context.Context, name string, obj runtime.Obje
 		}
 	}
 
+	// Set deletionOptions.Preconditions.ResourceVersion to ensure we're not
+	// racing with another PDB-impacting process elsewhere.
+	if deletionOptions.Preconditions == nil {
+		deletionOptions.Preconditions = &metav1.Preconditions{}
+	}
+	podRV := pod.ResourceVersion
+	deletionOptions.Preconditions.ResourceVersion = &podRV
```

Comment:
This should work, but it's causing the tests to fail locally for me.  I've spent some time debugging, etcd is incrementing the pod resource version when it gets it retrieves it to verify RV before the delete.  I'm assuming this is only impacting the local testing etcd, but I'll have more debug info coming in an issue soon.